libs/features/mas/web-components/src/variants/variant-layout.js

```diff
@@ -16,7 +16,7 @@ export class VariantLayout {
 
   constructor(card) {
     this.card = card;
-    setTimeout(() => this.insertVariantStyle(), 1);
+    this.insertVariantStyle();
```

Comment:
no perf impact then?

Reply:
I need to measure :)

Reply:
will attach some data.

Reply:
it is slower :) will investigate further.